jax/_src/debugger/core.py

```diff
@@ -113,19 +114,45 @@ def register_debugger(name: str, debugger: Debugger, priority: int) -> None:
 debug_lock = threading.Lock()
 
 
-def breakpoint(*, ordered: bool = False, backend=None, **kwargs):  # pylint: disable=redefined-builtin
-  """Enters a breakpoint at a point in a program."""
+def breakpoint(*, backend: Optional[str] = None, filter_frames: bool = True,
+               use_globals: bool = False, num_frames: Optional[int] = None,
+               ordered: bool = False, **kwargs):  # pylint: disable=redefined-builtin
+  """Enters a breakpoint at a point in a program.
+
+  Args:
+    backend: The debugger backend to use. By default, picks the highest priority
+      debugger and in the absence of other registered debuggers, falls back to
+      the CLI debugger.
+    filter_frames: Whether or not to filter out JAX internal frames.
```

Comment:
```suggestion
    filter_frames: Whether or not to filter out JAX-internal stack frames from the traceback. Since some libraries, like Flax, also make user of JAX's stack frame filtering system, this option can also affect whether stack frames from libraries are filtered.
```

WDYT? Or some rephrasing with similar meaning.